about.html

```diff
@@ -47,22 +47,33 @@ <h2>Technology</h2>
         <!-- begin Funding section -->
         <hr class="whitespace">
         <div class="row column text-center">
-            <h2>Funding and Project Participants</h2>
+            <h2><a class="anchor" id="funding"></a>Funding and Project Participants</h2>
         </div>
         <hr>
         <div class="row">
             <div class="small-10 small-centered medium-10 medium-centered columns">
                 <div class="row horizontal">
                     <div>
-                        <p>This project is funded by the BBSRC
-                        (<a href="https://bbsrc.ukri.org/research/grants/grants/AwardDetails.aspx?FundingReference=BB/R015384/1">BBR Award BB/R015384/1</a>)
-                        and is a collaboration between the
-                        <a href="https://www.openmicroscopy.org">OME Consortium</a> at
-                        Dundee (www.openmicroscopy.org) and EMBL-EBI.
-                        The IDR Project PIs are
+                        <p>The IDR Project PIs are
                         <a href="https://www.openmicroscopy.org/site/about/development-teams/jason">Jason Swedlow (Dundee)</a>,
                         <a href="https://www.ebi.ac.uk/about/people/alvis-brazma">Alvis Brazma (EMBL-EBI)</a> and
-                        <a href="https://research-information.bristol.ac.uk/en/persons/rafael-e-carazo-salas(a7638b29-53e4-49ba-82b5-98b21d82f41f).html">Rafael Carazo Salas</a>.</p>
+                        <a href="https://research-information.bristol.ac.uk/en/persons/rafael-e-carazo-salas(a7638b29-53e4-49ba-82b5-98b21d82f41f).html">Rafael Carazo Salas</a>.
+                        Construction of IDR project was funded by the BBSRC
+                        (Ref: <a href="https://bbsrc.ukri.org/research/grants/grants/AwardDetails.aspx?FundingReference=BB/M018423/1">BB/M018423/1</a>)
+                        as a collaboration between the
+                        <a href="https://www.openmicroscopy.org">OME Consortium</a> at Dundee
+                        and EMBL-EBI.  The IDR is part of several projects that have received
+                        funding from the European Union's Horizon 2020 research and innovation
+                        programme under grant agreements
+                        <a href="https://cordis.europa.eu/project/rcn/198825_en.html">688945</a>
+                        (<a href="http://www.eurobioimaging.eu/">Euro-BioImaging</a> Prep Phase II),
+                        <a href="https://cordis.europa.eu/project/rcn/198649_en.html">653493</a>
+                        (<a href="https://globalbioimaging.org/">Global BioImaging</a> Project) and
+                        <a href="https://www.cordis.europa.eu/project/rcn/197885_en.html">654248</a>
+                        (<a href="https://www.corbel-project.eu/home.html">CORBEL</a>). From 2018-2021, dataset
```

Comment:
maybe "Over 2018-2021" or "For 2018-2021" or "From 2018 to 2021"

Reply:
See 3c41ff3